packages/obonode/obojobo-chunks-math-equation/math-equation-properties-modal.js

```diff
@@ -112,6 +159,16 @@ class MathEquationProperties extends React.Component {
 					</div>
 					<span>{this.state.error}</span>
 				</div>
+				<div id="katexContainer" style={{ display: this.latexCheck(this.state.content.latex) }}>
+					<b id="previewTitle">Preview</b>
```

Comment:
For css classes/ids [use `kebab-case` instead of `camelCase`](https://github.com/ucfopen/Obojobo/wiki/Developer-Code-Style-Guide#class-naming-conventions)

Reply:
Same with 'katexContainer', 'katexPreview', and any others I might have missed